src/sidebar.cpp

```diff
@@ -36,6 +36,96 @@ static const trait_id trait_THRESH_FELINE( "THRESH_FELINE" );
 static const trait_id trait_THRESH_BIRD( "THRESH_BIRD" );
 static const trait_id trait_THRESH_URSINE( "THRESH_URSINE" );
 
+enum face_type : int {
+    face_human = 0,
+    face_bird,
+    face_bear,
+    face_cat,
+    num_face_types
+};
+
+std::string morale_emotion( const int morale_cur, const face_type face,
+                            const bool horizontal_style )
+{
+    if( horizontal_style ) {
+        if( face == face_bear || face == face_cat ) {
+            if( morale_cur >= 200 ) {
+                return "@W@";
+            } else if( morale_cur >= 100 ) {
+                return "OWO";
```

Comment:
Huh, didn't know that these were in the source now. Did #22318 land at some point?

Reply:
#22472 probably